src/main/java/TTT/controller/RestController.java

```diff
@@ -0,0 +1,19 @@
+package TTT.controller;
+
+import TTT.users.CustomUser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@org.springframework.web.bind.annotation.RestController
+public class RestController {
```

Comment:
Are you intending to add REST API for custom users or is it a leftover from studying?

Reply:
leftover from science. my oversight.